content/dev/languages/creating_a_language_pack.md

```diff
@@ -0,0 +1,291 @@
+
+### Admin Extras
+
+The Admin pages on zen cart hold a lot of information in the configuration table on the database. To do a complete conversion it will be necessary to convert the description fields to your chosen language. This will require a SQL patch.
+
+### Locations of language files
+
+These are the locations where you will find language files that may need translating.
+
+- emails **NOTE** Only required for v1.5.6 and earlier
+
+    - email_common.css
+
+    - email_template_checkout.htm
+
+    - more html files
+
+- admin
+
+    - includes
+
+        - languages
+
+            - english.php
+
+            - english
+
+               - admin_account.php
+
+
+               - more php files
+
+               - extra_definitions
+
+                   - ckeditor.php
+
+                   - more php files
+
+               - images
+
+                   - buttons
+
+                       - button_add_profile.gif
+
+                       - more gif files
+
+               - modules
+
+                   - newsletters
+
+                       - newsletter.php 
+
+                       - more php files
+
+- includes
+
+    - languages
+
+        - english.php
+
+        - english
+
+           - account.php
+
+           - more php files
+
+           - classic
+
+               - header.php
+
+           - extra_definitions
+
+               - cardinal3dsecure.php
+
+               - more php files
+
+               - classic
+
+                   - empty.txt
+               - responsive_classic
+
+                   - product_free_shipping.php
+
+           - html_incldues
+
+               - define_ask_a_question.php
+
+               - more php files
+
+               - classic
+
+                   - define_checkout_success.php
+
+                   - more php files
+
+               - responsive_classic
+
+                   - define_checkout_success.php
+
+                   - more php files
+
+           - images
+
+               - en_CA.gif
+
+               - more gif files
+
+           - modules
+
+               - order_total
+
+                   - ot_cod_fee.php
+
+                   - more php files
+
+                   - classic
+
+                       - empty.txt
+
+               - payment
+
+                   - authorizenet.php
+
+                   - more php files
+
+                   - classic
+
+                       - empty.txt
+
+               - shipping
+
+                   - flat.php
+
+                   - more php files
+
+                   - classic
+
+                       - empty.txt
+
+           - responsive_classic
+
+               - icon_names.php
+
+    - templates
+
+        - responsive_classic
+
+            - buttons
+
+                - english
+
+                    - button_update_cart.png
+
+        - template_default
+
+            - buttons

```

Comment:
These button images are ONLY relevant if the store has turned OFF the CSS buttons feature. No need to translate all the images in that case.

Same in responsive_classic

Reply:
I guess you did mention this up above. I'm sure others will miss it as I did, so probably still worth mentioning here, or completely skip the buttons dirs in this list, so it makes them come back and re-read the top when they discover the broken pattern.